src/view/com/util/MainScrollProvider.tsx

```diff
@@ -102,24 +83,15 @@ export function useOnMainScroll(): [OnScrollHandler, boolean, ResetCb] {
         startMode.value = mode.value
       }
     },
-    [headerHeight, mode, setMode, isScrolledDown, startDragOffset, startMode],
```

Comment:
Thanks to pulling this out, combined with https://github.com/bluesky-social/social-app/pull/2201, we'll finally stop re-rendering `Feed` when scrolling down/up:

<img width="919" alt="Screenshot 2023-12-13 at 06 01 47" src="https://github.com/bluesky-social/social-app/assets/810438/702da987-e699-4484-a1f9-ad918eefdf42">